src/features/ProfilePage/components/PasswordEditor.tsx

```diff
@@ -35,19 +40,21 @@ const PasswordEditor = () => {
     newPassword !== repeatedPassword;
 
   const isSavingDisabled =
-    !currentPassword.length || isPasswordTooShort || arePasswordsNotMatching;
+    isLoading ||
+    !currentPassword.length ||
+    isPasswordTooShort ||
+    arePasswordsNotMatching;
 
-  const saveNewPassword = () => {
-    console.log('API: Save new password');
-    setIsCurrentPasswordInvalid(true);
-  };
+  useEffect(() => {
+    if (isSuccess) setIsOpen(false);
+  }, [isSuccess]);
```

Comment:
hmm.. yeah maybe it is better to use the unwrapper.. wdyt?

Reply:
Changed! See my comment above!